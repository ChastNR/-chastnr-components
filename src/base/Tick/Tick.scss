.tick {
  height: 24px;
  transition-duration: 0.2s;
  transition-property: color, transform, -webkit-transform;
  transition-timing-function: ease;
  width: 24px;
  will-change: transform, color;
  color: #00b92d;
  animation: Cbj519JAt 0.225s;
}
