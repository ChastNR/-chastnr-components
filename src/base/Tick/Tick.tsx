import React from "react";

import { ReactComponent as TickSvg } from "./tick.svg";

import "./Tick.scss";

const Tick: React.FC = () => (
  <span className="tick">
    <TickSvg />
  </span>
);

export { Tick, TickSvg };
