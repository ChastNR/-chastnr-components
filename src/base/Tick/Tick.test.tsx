import { render } from "enzyme";
import toJson from "enzyme-to-json";

import { Tick } from "./Tick";

describe("<Tick/>", () => {
  it("should match snapshot", () => {
    const tick = render(<Tick />);
    expect(toJson(tick)).toMatchSnapshot();
  });
});
