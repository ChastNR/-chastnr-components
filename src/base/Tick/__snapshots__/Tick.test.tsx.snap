// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tick/> should match snapshot 1`] = `
<span
  class="tick"
>
  <span>
    content
  </span>
</span>
`;
